# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1504029729 14400
#      Tue Aug 29 14:02:09 2017 -0400
# Node ID b833c7334308dd4e17bb1c071c1c2784214a915a
# Parent  6f0c2963674ba128ecf1a6b876da61df354798db
* io.txi: Update note about scanf 'h' and 'l' modifiers.

diff --git a/doc/interpreter/io.txi b/doc/interpreter/io.txi
--- a/doc/interpreter/io.txi
+++ b/doc/interpreter/io.txi
@@ -964,18 +964,19 @@ could be read in as integers under the @
 these specifies a number with decimal value @code{10}.
 
 The @samp{%o}, @samp{%u}, and @samp{%x} conversions match unsigned
 integers in octal, decimal, and hexadecimal radices, respectively.
 
 The @samp{%X} conversion is identical to the @samp{%x} conversion.  They
 both permit either uppercase or lowercase letters to be used as digits.
 
-Unlike the C language @code{scanf}, Octave ignores the @samp{h},
-@samp{l}, and @samp{L} modifiers.
+By default, integers are read as 32-bit quantities.  With the @samp{h}
+modifier, 16-bit integers are used, and with the @samp{l} modifier,
+64-bit integers are used.
 
 @node String Input Conversions
 @subsection String Input Conversions
 
 This section describes the @code{scanf} input conversions for reading
 string and character values: @samp{%s} and @samp{%c}.
 
 The @samp{%c} conversion is the simplest: it matches a fixed number of
